include/bluetooth/mesh/light_ctl_cli.h

```diff
@@ -0,0 +1,391 @@
+			       struct bt_mesh_light_ctl_temp_status *rsp);
+
+/** @brief Set the CTL Temprature state of the server without requesting a
+ * response.
+ *
+ * @param[in] cli Client model to send on.
+ * @param[in] ctx Message context, or NULL to use the configured publish
+ * parameters.
+ * @param[in] set CTL Temprature state to set.
+ *
+ * @retval 0 Successfully sent the message.
+ * @retval -ENOTSUP A message context was not provided and publishing is not
+ * supported.
+ * @retval -EADDRNOTAVAIL A message context was not provided and publishing is
+ * not configured.
+ * @retval -EAGAIN The device has not been provisioned.
+ */
+int bt_mesh_light_ctl_temp_set_unack(
+	struct bt_mesh_light_ctl_cli *cli, struct bt_mesh_msg_ctx *ctx,
+	const struct bt_mesh_light_ctl_temp_set *set);
+
+/** @brief Get the Light CTL Default parameter state of the bound server.
+ *
+ * This call is blocking if the @c rsp buffer is non-NULL. Otherwise, this
+ * function will return.
+ *
+ * @param[in] cli Client model to send on.
+ * @param[in] ctx Message context, or NULL to use the configured publish
+ * parameters.
+ * @param[in] rsp Status response buffer, or NULL to keep from blocking.
+ *
+ * @retval 0 Successfully sent the message and populated the @c rsp buffer.
+ * @retval -EALREADY A blocking request is already in progress.
+ * @retval -ENOTSUP A message context was not provided and publishing is not
+ * supported.
+ * @retval -EADDRNOTAVAIL A message context was not provided and publishing is
+ * not configured.
+ * @retval -EAGAIN The device has not been provisioned.
+ * @retval -ETIMEDOUT The request timed out without a response.
+ */
+int bt_mesh_light_ctl_default_get(struct bt_mesh_light_ctl_cli *cli,
+				  struct bt_mesh_msg_ctx *ctx,
+				  struct bt_mesh_light_ctl_params *rsp);
+
+/** @brief Set the Default parameter state of the server.
+ *
+ * This call is blocking if the @c rsp buffer is non-NULL. Otherwise, this
+ * function will return.
+ *
+ * @param[in] cli Client model to send on.
+ * @param[in] ctx Message context, or NULL to use the configured publish
+ * parameters.
+ * @param[in] set Default parameters to set.
+ * @param[in] rsp Response status buffer, or NULL to keep from blocking.
+ *
+ * @retval 0 Successfully sent the message and populated the @c rsp buffer.
+ * @retval -EALREADY A blocking request is already in progress.
+ * @retval -ENOTSUP A message context was not provided and publishing is not
+ * supported.
+ * @retval -EADDRNOTAVAIL A message context was not provided and publishing is
+ * not configured.
+ * @retval -EAGAIN The device has not been provisioned.
+ * @retval -ETIMEDOUT The request timed out without a response.
+ */
+int bt_mesh_light_ctl_default_set(struct bt_mesh_light_ctl_cli *cli,
+				  struct bt_mesh_msg_ctx *ctx,
+				  const struct bt_mesh_light_ctl_params *set,
+				  struct bt_mesh_light_ctl_params *rsp);
+
+/** @brief Set the Default parameter state of the server without requesting a
```

Comment:
Perhaps "default CTL value" instead of "Default parameter state" is better? Should be changed throughout in that case.